### PR TITLE
Added Travis CI integration for Python tests

### PR DESCRIPTION
**Pull request by sumnerevans**

### CODE REVIEW

File: server/helpers/util.py [hunk 1]
```diff
@@ -2,14 +2,11 @@
...
 import json
-import yaml
-import helpers
 
 from datetime import datetime
-from markupsafe import Markup
-
-
 
+import helpers
```
Comment (ReeceHughes) [inline server/helpers/util.py]: Any reason why the imports were re-arranged?
Also, remove newlines.

File: server/tests/parser_tests.py [hunk 2]
```diff
@@ -3,10 +3,11 @@
 """
 
 import json
-import parsing_lib
 
 from datetime import datetime
 
+import parsing_lib
```
Comment (ReeceHughes) [inline server/tests/parser_tests.py]: Again with the re-arranged imports and newlines.

File: server/helpers/util.py (hunk 1 above)
Comment (sumnerevans) [inline server/helpers/util.py]: imports from the standard library should be before imports from non-standard libraries

Comment (sumnerevans) [inline server/helpers/util.py]: Newlines are to separate imports from import-froms

File: server/helpers/util.py [hunk 3]
```diff
@@ -2,14 +2,11 @@
...
+import yaml
```
Comment (cnavrides) [inline server/helpers/util.py]: should be two lines before the def

File: server/helpers/util.py [hunk 4]
```diff
@@ -2,14 +2,11 @@
 Utility Functions
 """
 import json
-import yaml
-import helpers
```
Comment (cnavrides) [inline server/helpers/util.py]: No newline here: https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting

File: server/tests/parser_tests.py [hunk 5]
```diff
@@ -3,10 +3,11 @@
 """
 
 import json
-import parsing_lib
```
Comment (cnavrides) [inline server/tests/parser_tests.py]: I believe no newline here

**Pull request merged**